README.rst

```diff
@@ -424,6 +424,17 @@ TMT_REBOOT_COUNT
     for requesting the reboot, variables ``REBOOTCOUNT`` and
     ``RSTRNT_REBOOTCOUNT`` contain number of reboots as well.
 
+TMT_GUEST_HOSTNAME, TMT_GUEST_ROLE
+    The hostname of the guest on which the test is running, and
+    the multihost role of the guest, if it was specified in plan's
+    ``provision`` step, via ``role`` key.
+
+TMT_ROLE_*
+    Space-separated list of guests of a given role. Each known role
+    spawns a variable, listing the corresponding guests. For example,
+    for guests with ``role: foo``, ``TMT_ROLE_FOO=guest1 guest2 ...``
```

Comment:
Stating obvious but from the last hacking session it seems we decided to  keep role name as is, thus `TMT_ROLE_foo` will be created.